### PR TITLE
adding goimports as a make target

### PR DESCRIPTION
**Pull request by sbawaska**
Using goimports to format the source and organize imports.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,12 @@ build: kab
 test:
 	GO111MODULE=on go test ./... -race -coverprofile=coverage.txt -covermode=atomic
 
+check-goimports:
+	@which goimports > /dev/null || (echo goimports not found: issue \"GO111MODULE=off go get golang.org/x/tools/cmd/goimports\" && false)
+
+goimports: check-goimports
```
[the remark this answers is not in the document]
Comment (sbawaska) [reply Makefile]: pushed a change for this.

File: Makefile [hunk 2]
```diff
@@ -13,6 +13,12 @@ build: kab
...
+	sh -c "find . -name \"*.go\" | fgrep -v vendor/ | fgrep -v pkg/apis | xargs goimports -w -local github.com/projectriff/cnab-k8s-installer-base"
```
[the remark this answers is not in the document]
Comment (sbawaska) [reply Makefile]: the packages are now whitelisted.

File: Makefile [hunk 3]
```diff
@@ -6,13 +6,22 @@ VERSION ?= $(shell cat VERSION)
...
-all: build test
+all: build test verify-goimports
 
 build: kab
 
 test:
 	GO111MODULE=on go test ./... -race -coverprofile=coverage.txt -covermode=atomic
 
+check-goimports:
+	@which goimports > /dev/null || (echo goimports not found: issue \"GO111MODULE=off go get golang.org/x/tools/cmd/goimports\" && false)
```
Comment (scothis) [inline Makefile]:
```suggestion
	@which goimports > /dev/null || (echo goimports not found: issue \"go get golang.org/x/tools/cmd/goimports\" && false)
```

File: pkg/kab/resource_manager_test.go [hunk 4]
```diff
@@ -17,16 +17,17 @@
...
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/projectriff/cnab-k8s-installer-base/pkg/apis/kab/v1alpha1"
+	"github.com/projectriff/cnab-k8s-installer-base/pkg/kab"
+	"github.com/projectriff/cnab-k8s-installer-base/pkg/kab/vendor_mocks"
+	mockkubectl "github.com/projectriff/cnab-k8s-installer-base/pkg/kubectl/mocks"
 	"github.com/stretchr/testify/mock"
 	v12 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (scothis) [inline pkg/kab/resource_manager_test.go]: `metav1` is a better import name. Along the same lines `v12` should be `corev1`

Comment (sbawaska) [reply pkg/kab/resource_manager_test.go]: fixed.

**Pull request merged**